### PR TITLE
Add py-pot with patch

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

**Review by bernhardkaindl — DISMISSED** [review]

**Review by alalazo — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-pot/package.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    problems related to Optimal Transport for signal, image processing and
+    machine learning.
+    """
+
+    homepage = "https://github.com/PythonOT/POT"
+    pypi     = "POT/POT-0.7.0.tar.gz"
+
+    version('0.7.0', sha256='d4ac2bc8791f049a3166820d51e218d6c299885449b735eafef8d18c76d4ad06')
+
+    # They're setting CC env variables :/
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-pot/package.py]: Can you rephrase this comment and briefly explain when CC is removed / added ?

**Review by alalazo — CHANGES_REQUESTED** [review]
Basically LGTM. Only a minor request to rephrase a comment.

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
@spackbot run pipeline
